design-patterns/domain-driven-design/pattern1/README.md

```diff
@@ -0,0 +1,194 @@
+# Pattern 1: Single Domain, Single Bounded Context, and a Single Uni
+
+
+## Scenario
+
+The Customer Relationship Management(CRM) Team wants to create a single source of truth for customer account data, which includes firstname, lastname, customerId, personal details, contact details, and the address. The CRM Team will frequently query customer data using customerId. 
+
+
+## Modeling
+
+Since CRM is an independent business unit we will assume it is a domain on its own. Inside CRM, we will have multiple sub-domains; customer account is one such sub-domain. You will model customer account as a Bounded Context.  Since CRM will perform frequent lookup, search, and filter on customer records by customerId, you will index customer accounts on the customerId property. 
+
+If you are confused between Bounded Context, Entity, and the Aggregate. These terms correlate 1:1 for the purpose of this exercise. I will use the term Entity here to mean Root Entity which is an Aggregate in DDD. And, Bounded Context can have one or more than one Entity.
+
+
+
+
+## Create a Uni
+
+1. Login to [Vendia Share](https://share.vendia.net/login). 
+
+Don’t have a login, you can [sign up](https://share.vendia.net/) now. 
+
+2. Create Uni
+
+Click the `Create Universal Application` button on the top-right. Next, use the `Create your own` option. 
+
+3. Fill out details in the Uni creation wizard. 
+
+1. **Step 1:** Give your Uni a name. 
```

Comment:
The numbering here is losing me.  1, 2, 3, 1, 2, 3, 4.

There's also some odd character color coding happening below.  Not sure if this is just a GitHub issue or if there's something syntactically invalid along the way.